x-pack/platform/plugins/shared/alerting/server/alerts_service/lib/update_index_template_fields_limit.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import type { ElasticsearchClient } from '@kbn/core/server';
+import type { IndicesGetIndexTemplateIndexTemplateItem } from '@elastic/elasticsearch/lib/api/types';
+
+export const updateIndexTemplateFieldsLimit = ({
+  esClient,
+  template,
+  limit,
+}: {
+  esClient: ElasticsearchClient;
+  template: IndicesGetIndexTemplateIndexTemplateItem;
+  limit: number;
+}) => {
+  return esClient.indices.putIndexTemplate({
+    name: template.name,
+    body: {
+      ...template.index_template,
+      // @ts-expect-error elasticsearch@9.0.0 https://github.com/elastic/elasticsearch-js/issues/2584
```

Comment:
We should be able to remove this workaround: https://github.com/elastic/kibana/pull/217216

Reply:
ok I had to remove the `body` wrapper as well. ok now.